src/libraries/System.Text.Json/tests/Common/JsonNumberTestData.cs

```diff
@@ -237,6 +247,58 @@ static JsonNumberTestData()
             }
             #endregion
 
+#if NETCOREAPP
+            #region generate Int128s
+            Int128s = new List<Int128>
+            {
+                0,
+                (Int128)long.MaxValue + 1,
+                (Int128)long.MinValue - 1,
+                Int128.MaxValue,
```

Comment:
Do we have any tests checking what happens when you try to deserialize a JSON number equalling `Int128.MaxValue + 1` and `Int128.MinValue - 1`?

Reply:
We would need some kind of testing that validates handling of payloads with unsupported JSON numbers such as numbers exceeding `Int128.MaxValue` and floating point numbers.